### PR TITLE
applications: nrf_desktop: Align documentation due to upmerge

### PR DESCRIPTION
**Pull request by alstrzebonski**

### CODE REVIEW

File: applications/nrf_desktop/description.rst [hunk 1]
```diff
@@ -327,7 +327,7 @@ Depending on the development kit you use, you need to select the respective conf
 
       .. table-from-rows:: /includes/sample_board_rows.txt
          :header: heading
-         :rows: nrf52840gmouse_nrf52840
+         :rows: nrf52840gmouse/nrf52840
```
Comment (gmarull) [inline applications/nrf_desktop/description.rst:330]: this is wrong, have you tried building docs?

**Review by gmarull — CHANGES_REQUESTED** [review]
please do not change `:rows:` content, this still works using underscores for now.

File: applications/nrf_desktop/description.rst (hunk 1 above)
Comment (alstrzebonski) [reply applications/nrf_desktop/description.rst]: Sorry, didn't know that. I've dropped this change.

**Review by kapi-no — APPROVED** [review]

**Comment by gmarull** [comment]
integrated

**Pull request closed without merging**